testing/BUILD.gn

```diff
@@ -220,3 +191,38 @@ if (enable_unittests) {
     }
   }
 }
+
+# All targets on all platforms should be able to use the Metal utilities. On
+# platforms where Metal is not available, the tests must be skipped or
+# implemented to use another available client rendering API. This is usually
+# either OpenGL which is portably implemented via SwiftShader or the software
+# backend. This way, all tests compile on all platforms but the Metal backend
+# is exercised on platforms where Metal itself is available.
+#
+# On iOS, this is enabled to allow for Metal tests to run within a test app
+if (enable_unittests || is_ios) {
+  source_set("metal") {
+    if (shell_enable_metal) {
```

Comment:
So this was just moved from inside the overall `enable_unittests` block into its own block that also allows for iOS, so it was already like that (except for the iOS condition).

I am a little wary of poking this because the large comment directly above the `source_set` suggests that there's some stuff going on here that's not entirely straightforward.